src/components/file-input/README.md

```diff
@@ -1 +1,99 @@
-# File input
\ No newline at end of file
+# File input
+
+## When to use
+Use the file input component when the user is required to upload a file from their local computer.
+
+## Adding the Markup
```

Comment:
I think this section has become a little too divided. More info below for details.